### ISSUE TITLE
Github login credentials stopped working

### ISSUE THREAD

**Issue by kco1206** [top2429/Finance-App#5]
Issue Type

Bug

**Description**

Suddenly my github login stopped working today. It seem to work fine yesterday. I can login my github account directly in git but not through VS Code. Please help! Kindly take note that TFA is activated to my account (something you need to consider during troubleshooting).

**VS Code Info**

VS Code version: Code 1.20.1 (https://github.com/microsoft/vscode/commit/f88bbf9137d24d36d968ea6b2911786bfe103002, 2018-02-13T15:34:36.336Z)
OS version: Windows_NT x64 6.1.7601

**Comment by kco1206** [comment]
I'm sure that GitHub's credential manager is a very nice thing,

From my innocent/naive/newb point of view it was VS code that F'd up.

Proof of the pudding is that that I came to the VS Code GitHub presence to try to resolve the issue. And - lo and behold - it was resolved here.

Therefore in order to prevent other newbs from having issues it might be a good thing to record the tip from @CoenraadS where people can see it.

**Comment by top2429** [comment]
PROBLEM SOLVED: I followed the suggested link https://github.com/git-for-windows/git/releases to update the credential manager. The link didn't have anything about the credential manager, but I installed git for windows from that link and it solved the problem. Presumably git for windows installs a credential manager (although I'm a little confused because I thought credential manager was a windows app).
